tools/ci/marshalcheck/main/main.go

```diff
@@ -0,0 +1,10 @@
+package main
```

Comment:
structure wise I think you can put `main.go` in the root (and probably rename the tool):
```
- tools/ci/api-linter/
  - marshalcheck/
    - marshalcheck.go
    - marshalcheck_test.go
    - suite_test.go
  - main.go
```